templates-generator/index.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs');
+const buildOptions = require('minimist-options');
+const path = require('path');
+const glob = require('glob');
+const micromatch = require('micromatch');
+const args = require('minimist')(process.argv.slice(2), buildOptions({
+    platform: {
+        type: 'string',
+        alias: 'p'
+    }
+}));
+
+const platformsConfigs = {
+    react: './react-config.js'
+};
+
+const commands = args['_'];
+(() => {
+    if(commands.length) {
+        console.error('Command is redundant');
+        return;
```

Comment:
I suggest to exit with not success  code in this case

Reply:
Fix